src/components/SingleReview.js

```diff
@@ -20,6 +20,11 @@ export const SingleReview = () => {
     patchReviewById(review_id);
   };
 
+  const handleDeductVoteClick = () => {
+    setCount((currCount) => currCount - 1);
```

Comment:
Deduct vote decrements client side, but actually increments in the database. 
If you refresh after deducting some, its actually increased the database votes